hangman/lib/hangman.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Hangman do
+
+  defstruct(
+    game_state: :initializing,
+    turns_left: 7,
+    word: "",
+    guess: "",
+    used: [],
```

Comment:
I wonder if you need all three of these lists? I'm thinking you could probably make to with a set of the letters guessed, and a list containing the letters in the word. The value for the word passed back in the tally can be calculated on the fly from these, without adding to the state.